docs/bootstrap/cloud-config.tpl

```diff
@@ -0,0 +1,27 @@
+#cloud-config
```

Comment:
@jsiebens had you deleted this file? I'm doing some upgrades using an older branch as a basis.

Reply:
Yes I think so. When changing the cloud init files to use the hack/install.sh script, I noticed that every file in docs/bootstrap was duplicated in docs/bootstrap/digitalocean-terraform. So I removed the file you mention (and others) and kept the files in docs/bootstrap/digitalocean-terraform